### DISCUSSION TITLE
How connect to the Group and react on questions by specific words like "How..."?

### DISCUSSION THREAD

**Discussion by chicago666666** [n3d1117/chatgpt-telegram-bot#56] [Q&A] [answered]
How connect to the Group and react on questions by specific words like "How..."?
I have connected bot to the group and thought it would start participating in general chat 😄 Is that possible? 🤔

**Comment by n3d1117** [comment, marked as answer]
Hi @chicago666666, you can use the bot in a group by writing an inline message like this:

`@your_bot_name How ...`

Make sure to enable inline mode via BotFather, using the `/setinline` command. 

Then click on the inline prompt that appears and the bot should reply to your question. From there on, you can also reply to a bot's message to get an answer.

If you instead want the bot to react to every incoming message in the group chat, you need to disable privacy mode from BotFather, and optionally set `GROUP_TRIGGER_KEYWORD` as prefix you'd like to use to address the bot.

Let me know if it helps!
